### PR TITLE
Enhancements for infinite-scroll

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/1598

Fixes following problems:

- [x] Click on image (masonry) single select - although other images selected
- [x] Multiple edit only with edit button
- [x] Limit of pagination separated (dropdown and infinite)
- [x] Infinite-scroll limit 50 cannot be changed
- [x] List-toolbar should be on the top of page like tabs
- [x] Show only selected in datagrid (link for selected-counter)

### CODE REVIEW

**Comment by danrot** [comment]
StyleCI check is failing

**Comment by danrot** [comment]
Ah sry, just realized that this PR is not in review yet.

**Comment by wachterjohannes** [comment]
@danrot now this should be ready to review!

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js [hunk 1]
```diff
@@ -337,6 +362,34 @@ define(['services/sulumedia/media-manager',
...
             this.sandbox.emit('husky.dropzone.' + this.options.instanceName + '.enable');
+        },
+
+        /**
+         * loads the collection-data into this.data. is automatically executed before component initialization
+         * @returns {*}
+         */
+        loadComponentData: function() {
+            var promise = this.sandbox.data.deferred();
+            CollectionManager.loadOrNew(this.options.id, UserSettingsManager.getMediaLocale()).then(function(data) {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js]: Why don't we just return the promise from `loadOrNew`, but create a new one instead?

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js]: your right i will change that (:

**Comment by wachterjohannes** [comment]
@danrot i will build the bundles and translations when this branch is merged in the base branch

**Comment by danrot** [comment]
I've encountered two more things: 

1. The link to show only the selected assets in the masonry view should get a pointer cursor.
2. When changing from list to masonry view the data is loaded too late (so you see the one page for a few seconds, and then its changed to the first page)

**Comment by wachterjohannes** [comment]
i have fixed this two comments from @danrot but we should display a loader somewhere when we change the view /cc @berndhep we should look at this on monday

**Pull request merged**